projects/fsastorefrontstyles/scss/base/fs-accordions.scss

```diff
@@ -33,7 +33,7 @@
     @include fs-box-shadow(0, 8px, 6px, -6px, $color-grey, 1);
     position: relative;
     .fs-icon,
-    .fas {
+    .fas-accordion {
```

Comment:
Why change this? Seems redundant, and this way all icons used in the accordion that are not our custom `fs-icon`, would have to have `fas-accordion` class. With `.fas` as it was before all font awesome icons are covered, right?